os/arch/arm/src/amebad/amebad_idle.c

```diff
@@ -88,76 +88,40 @@
  ****************************************************************************/
 
 /****************************************************************************
- * Name: up_idlepm
+ * Name: up_pm_board_sleep
  *
  * Description:
  *   Perform IDLE state power management.
  *
+ * Input Parameters:
+ *   handler - The handler function that must be called after each board wakeup.
+ *
+ * Returned Value:
+ *   None.
+ *
  ****************************************************************************/
 
 #ifdef CONFIG_PM
-static void up_idlepm(void)
+void up_pm_board_sleep(void (*handler)(clock_t, pm_wakeup_reason_code_t))

```

Comment:
is void (*handler)(clock_t, pm_wakeup_reason_code_t) this better than void *handler?

Reply:
Yes, its more readable